libs/feature-flags/src/lib/features.ts

```diff
@@ -29,6 +29,11 @@ export enum Features {
 
   // Application System Delegations active
   applicationSystemDelegations = 'applicationSystemDelegations',
+
+  // Service portal modules
```

Comment:
Good change 👍 

One question, would it make sense to split these into separate files where each project has it's own `features.ts` file?

for exaple service-portal would go into `libs/feature-flags/src/lib/features-service-portal.ts` or some variation of that. And the codeowners for the service portal would be set as codeowners on that file as well.

My thinking is, so that we avoid always needing a code review from core just to add a feature flag string variable, and less noise for the core team as well.

Reply:
I don't mind the noise, but we can do it if it's causing delays or annoyances.

My main thinking is that many of the service-portal flags we're creating shouldn't just affect the UI. They should be checked on the backend as well, eg for data integrations which are "not ready". Putting them in a "service-portal-features" file or having service-portal in the name of the field can cause mistakes in this regard.

Reply:
That makes sense, I don't think we're adding too many feature flagged modules anyways. 💯